docs/docs/api/02-Commands/swipe.md

```diff
@@ -5,5 +5,5 @@ displayed_sidebar: apiSidebar
 
 Swipe an element in the x and y direction
 
-   * @param {number} x_offset - A x direction. positive an negative values are accepted
-   * @param {number} y_offset - A y direction. positive an negative values are accepted
+   * @param {number} x_offset - A x direction. positive and negative values are accepted
+   * @param {number} y_offset - A y direction. positive and negative values are accepted
```

Comment:
Did you changed all the spell correction inside the askui-inference repro, too? Otherwise they get overwritten

Reply:
Yes: See https://github.com/askui/askui-inference/pull/73

On another note. Is the documentation automatically overridden? I want to add more docs (gifs, extended examples to each command). I do not think inference would be the right place to hold it, or is it?

Reply:
The API folder gets overwritten by any changes of the dsl in askui-inference. --> The changes should be adapted inside the askui-inference. 

The documentation is used inside the esdoc (inline code documentation) and the docs (docusaurus). If you add some gifs, we have to check if the in-line doc can display them too. 

We should have a talk